src/components/EmojiPicker/index.tsx

```diff
@@ -0,0 +1,71 @@
+import {SFC} from 'types';
+import {useEffect, useRef, useState} from 'react';
+import {mdiEmoticonOutline} from '@mdi/js';
+import Icon from '@mdi/react';
+import EmojiPicker from 'emoji-picker-react';
+import * as S from './Styles';
+
+interface EmojiBoxProps {
+  field: string;
+  value: string;
+  setFieldValue: (field: string, value: string) => void;
+}
+const EmojiBox: SFC<EmojiBoxProps> = ({field, value, setFieldValue}) => {
```

Comment:
According to the above ordering
```suggestion
const EmojiBox: SFC<EmojiBoxProps> = ({field, setFieldValue, value}) => {
```